lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -15,6 +15,7 @@ class AmazonMarketplaceWebService < Service
     VERSION = "2010-10-01"
 
     SUCCESS, FAILURE, ERROR = 'Accepted', 'Failure', 'Error'
+    XML_FAILURE_RESPONSE = { :success => FAILURE }.freeze
```

Comment:
I'd prefer to just freeze this inline rather than pulling it into a constant. Do we still get the performance benefit?